edb/pgsql/metaschema.py

```diff
@@ -3107,11 +3107,13 @@ def __init__(self) -> None:
                 'postgres default',
                 'postgres environment variable',
                 'postgres configuration file',
+                'environment variable',
+                'command line',
                 'postgres command line',
                 'postgres global',
+                'postgres client',
                 'system override',
                 'database',
-                'postgres client',
```

Comment:
I think `postgres client` was after `system override` and `database` because we don't want users to override our server-specific postgres config set at connection time like `client_encoding` and `timezone`. But with this PR on particularly remote Postgres, we use the same `postgres client` config for those `backend_setting` set in command-line/env, they are meant to be override by `system override` or `database`. Because we didn't expose `client_encoding` or `timezone` as EdgeDB config, I hope it is fine to change the priority of `postgres client`.